app/src/main/java/ca/llamabagel/transpo/trips/ui/StopViewModel.kt

```diff
@@ -0,0 +1,30 @@
+package ca.llamabagel.transpo.trips.ui
+
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.ViewModel
+import androidx.lifecycle.viewModelScope
+import ca.llamabagel.transpo.data.db.StopId
+import ca.llamabagel.transpo.trips.domain.GetNextBusTripsUseCase
+import ca.llamabagel.transpo.trips.ui.adapter.TripAdapterItem
+import kotlinx.coroutines.flow.collect
+import kotlinx.coroutines.launch
+import javax.inject.Inject
+
+class StopViewModel @Inject constructor(
+    private val getNextBusTrips: GetNextBusTripsUseCase
+) : ViewModel() {
```

Comment:
This can also fit on 1 line... Is the auto formatter doing this??

Reply:
Probably